humility-core/src/hubris.rs

```diff
@@ -434,23 +454,25 @@ macro_rules! dump {
                     p_type: $program_hdr::PT_LOAD,
                     p_flags: $program_hdr::PF_R,
                     p_offset: offset,
-                    p_vaddr: region.base,
-                    p_filesz: region.size,
-                    p_memsz: region.size,
+                    // TODO these cast are only needed until hubris::regions has 64bit support
```

Comment:
Once the `regions.base` and `region.size` are `$abi_size` clippy should yell at us for the cast (not passing CI), so I figured no need to track this separately.  Also, it would not break anything to leave it in, just some extra syntax.